cardigan_beta/stories/wip/BookPromo.js

```diff
@@ -0,0 +1,23 @@
+import { storiesOf } from '@storybook/react';
+import { withReadme }  from 'storybook-readme';
+import { text } from '@storybook/addon-knobs/react';
+import { url, image, singleLineOfText } from '../content';
+import BookPromoReadme from '../../../common/views/components/BookPromo/README.md';
+import BookPromo from '../../../common/views/components/BookPromo/BookPromo';
+
+const BookPromoExample = () => {
+  return (
+    <BookPromo
+      url={url}
+      image={image('https://iiif.wellcomecollection.org/image/prismic:1e958377a9f21d49a5de6578212e02ad4381d473_9781781254875_0.png/full/full/0/default.png')}
+      title={text('Title', singleLineOfText(2, 6))}
+      subTitle={text('Sub title', singleLineOfText(3, 6))}
+      description={text('Description', singleLineOfText(10, 20))}
+    />
+  );
+};
+
+const stories = storiesOf('WIP', module);
+
+stories
+  .add('Promos / Book promo', withReadme(BookPromoReadme, BookPromoExample));
```

Comment:
We'll need to make sure the prefix matches the name above ☝️ .
Maybe I take the above back, could we call it (`Components (WIP)`) as then we know we have a starting point (I was previously a bit harsh)